.changes/e1fa257d-73b3-4616-8c39-f32956008124.json

```diff
@@ -0,0 +1,5 @@
+{
+    "id": "e1fa257d-73b3-4616-8c39-f32956008124",
+    "type": "bugfix",
```

Comment:
I question whether this is a bugfix, but I wouldn't say it's exactly a feature either...

Reply:
I also struggled with this and ultimately decided on **bugfix** because I consider the placement of the DSL overloads in the to be a "design bug".

But it's not a perfect fit. Our options are **bugfix**, **feature**, **documentation**, or **miscellaneous** which I generally reserve for chores. Does **miscellaneous** fit better here?

Reply:
Perhaps we're missing a "refactor" category?